clarity/contracts/flash-loan-user-margin-usda-wbtc.clar

```diff
@@ -3,15 +3,18 @@
 
 (define-constant ONE_8 (pow u10 u8))
 
-(define-public (execute (collateral <ft-trait>) (amount uint) (memo (optional uint)))
+(define-public (execute (collateral <ft-trait>) (amount uint) (memo (optional (buff 16))))
     (let
         (   
             ;; gross amount * ltv / price = amount
             ;; gross amount = amount * price / ltv
-            (ltv (try! (contract-call? .collateral-rebalancing-pool get-ltv .token-usda .token-wbtc (default-to u0 memo))))
-            (price (try! (contract-call? .yield-token-pool get-price (default-to u0 memo) .yield-usda)))
+            ;; buff to uint conversion
+            (memo-uint (buff-to-uint (unwrap-panic memo)))
```

Comment:
Instead of `unwrap-panic`, please handle and throw a more informative error if `memo` is `none`.